src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,20 @@
+package duke;
+
+import javafx.application.Application;
+
+/**
+ * Main class of Duke
```

Comment:
Should this have a punctuation after it?

Reply:
yes it does look a bit odd.

Reply:
Yes, you're right, I missed that. Thanks for helping me spot the mistake!